examples/tensorflow/text-classification/run_text_classification.py

```diff
@@ -119,6 +60,56 @@ def on_epoch_end(self, epoch, logs=None):
         self.model.save_pretrained(self.output_dir)
 
 
+def convert_dataset_for_tensorflow(dataset, non_label_column_names, batch_size, labels, dataset_mode, drop_remainder):
+    """Converts a Hugging Face dataset to a Tensorflow Dataset or Sequence object. We usually only want a Dataset when
+    we're training on TPU, as we get the nice feature of variable-length batches when we put the data in a Sequence
+    object instead."""
```

Comment:
For the docstrings on multiple line we use the following:
```suggestion
    """
    Converts a Hugging Face dataset to a Tensorflow Dataset or Sequence object. We usually only want a Dataset when
    we're training on TPU, as we get the nice feature of variable-length batches when we put the data in a Sequence
    object instead.
    """
```

Reply:
It also looks like we never return a `Sequence`?

Reply:
You're right - this text has been updated